msal/application.py

```diff
@@ -1552,10 +1552,26 @@ def _acquire_token_silent_from_cache_and_possibly_refresh_it(
                     correlation_id=correlation_id,
                     auth_scheme=auth_scheme,
                     **data)
+                
+                if (force_refresh and response.get("access_token")):
+                        at_to_renew = response.get("access_token")
```

Comment:
I supposed this is a functioning tactical solution which wires up "native MSAL"'s `force_refresh` programming model with MSAL C++'s `access_token_to_renew` model. My only hesitation is that this PR further exposes/emphasizes the `force_refresh` which risks being abused.

Let me CC @bgavrilMS to hear his thoughts on whether we take this opportunity to consider exposing the `access_token_to_renew` pattern in MSAL (and then deprecate the `force_refresh`).

Afterwards, I'll get back to @Ugonnaak1 on how to polish this PR, especially on adding test cases.

Reply:
I think it's ok to support `force_refresh` since it's already there on MSALs. If we decided to deprecate it, we should do it all at once.

Reply:
@rayluo - let's please use `force_refresh` as boolean. We will figure out a migration path if needed. There is telemetry on all MSALs that supposed to tell us how often this is used.